packages/icons/src/cloud.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+import { IconProps } from "./types";
+
+export const CloudIcon = React.forwardRef<SVGSVGElement, IconProps>(
```

Comment:
this one looks like made by @taylornowotny since its 16, but lets check if its not one of https://icons.radix-ui.com/ 

maybe we need to have a way to communicate which icon is from where for now